apiserver/src/main/resources/migration/procedures/procedure_migrate_metrics_to_partitions.sql

```diff
@@ -0,0 +1,53 @@
+CREATE OR REPLACE PROCEDURE "MIGRATE_METRICS_TO_PARTITIONS"(
+    target_table TEXT,
+    source_table TEXT
+)
+LANGUAGE "plpgsql"
+AS
+$$
+DECLARE
+    retention_days INTEGER := 90;
+    start_date DATE;
+    end_date DATE := current_date;
```

Comment:
We could take this opportunity to create the next partition already:

```suggestion
    end_date DATE := current_date + '1 day'::interval;
```